CHANGELOG.next.asciidoc

```diff
@@ -111,6 +111,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Skipping unparsable log entries from docker json reader {pull}12268[12268]
 - Require client_auth by default when ssl is enabled for tcp input {pull}12333[12333]
 - Require certificate authorities, certificate file, and key when SSL is enabled for the TCP input. {pull}12355[12355]
+- Update CoreDNS module README file to reflect the changes to autodiscover default input settings. {pull}12382[12382]
```

Comment:
probably you don't need this entry as the previous change hasn't been released yet

Reply:
@exekias Thanks! Do I need to manually backport this to 7.2? Or will it be handled automatically?